specs/mediawiki/v1/content.yaml

```diff
@@ -1142,10 +1143,17 @@ paths:
           type: integer
           required: true
         - name: sections
-          in: formData
+          in: body
           description: Sections to transform
           type: string
           required: true
+          schema:
+            type: 'object'
+            properties:
+              sections:
```

Comment:
Given how random those IDs are currently, I would be surprised if anybody was using the entry point. It's officially marked as unstable, which means that it might change with fairly quickly.